t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -305,7 +306,7 @@ func (d *groupChatParams) Encode() (string, error) {
 
 //Start status node
 func (s *WhisperMailboxSuite) startBackend(name string) (*api.StatusBackend, func()) {
-	datadir := "../../.ethereumtest/mailbox/" + name
+	datadir := filepath.Join(RootDir, ".ethereumtest/mailbox/") + name
```

Comment:
`filepath.Join(RootDir, ".ethereumtest/mailbox", name)` should work the same.

Reply:
+1